### PR TITLE
[OYPD-289] Add membership demo content & demo join page.

### PR DESCRIPTION
**Pull request by Kerby70**
Jira https://propeople-us.atlassian.net/browse/OYPD-289
D.o https://www.drupal.org/node/2859341

Steps to review:
- [x] See main menu link to "Join" page
- [x] See footer right menu link to "Join" page
- [x] Visit /join
- [x] See title "Give the Y a Try", and gallery
- [x] See membership calculator with 7 membership types & each with people graphics

1. UNRELATED Feature: Static Paragraphs "Scenario: Create Small Banner", fails on `When I press "Add Small Banner" in the "header_area" ` with the message `(The button 'Add Small Banner' was not found in the region 'header_area' on the page http://openy-dev.ffwua.com/build709/node/add/landing_page)`. This test failure was missed on https://github.com/ymcatwincities/openy/pull/215 for https://propeople-us.atlassian.net/browse/OYPD-293 which removed the small banner from the header content field options.
image

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
<strong>2</strong>  errors for CodeSniffer: Drupal standard file http://openy-dev.ffwua.com/build714/build_reports/Drupalsniff.txt
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build714 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build714 |
| Installation process | http://install.openy-dev.ffwua.com/build714/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/714/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build714/profiles/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build715 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build715 |
| Installation process | http://install.openy-dev.ffwua.com/build715/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/715/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build715/profiles/openy/build/reports/behat

**Comment by Kerby70** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build723 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build723 |
| Installation process | http://install.openy-dev.ffwua.com/build723/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/723/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build723/profiles/openy/build/reports/behat

File: modules/openy_features/openy_demo_content/modules/openy_demo_nmbrshp/config/install/migrate_plus.migration.node_membership.yml [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  data_rows:
+    -
+      id: couple
+      title: 'Couple'
+      description: |
+        <p>Two adults in&nbsp;same household</p>
+
+        <p>Join Fee: $125&nbsp;</p>
+
+        <p>Monthly Rate: $120</p>
```
Comment (alexschedrov) [inline modules/openy_features/openy_demo_content/modules/openy_demo_nmbrshp/config/install/migrate_plus.migration.node_membership.yml]: Please remove `Join Fee` and `Monthly Rate` from all descriptions and specify it in the membership type fields.
image

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build728 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build728 |
| Installation process | http://install.openy-dev.ffwua.com/build728/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/728/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build728/profiles/openy/build/reports/behat

**Comment by AlexNetman** [comment]
Why I don't see Alerts on the latest build?

**Comment by alexschedrov** [comment]
@AlexNetman because it hasn't been merged with master yet.

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build731 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build731 |
| Installation process | http://install.openy-dev.ffwua.com/build731/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/731/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build731/profiles/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**